src/lib/ts/cache-compiler-host.ts

```diff
@@ -88,19 +89,6 @@ export function cacheCompilerHost(
       return cache.content;
     },
 
-    // ng specific
-    moduleNameToFileName: (moduleName: string, containingFile: string) => {
```

Comment:
Oh, it's gone from here. Ah, there's actually two copies of the compiler host and the legacy one was updated. I am surprised this method was here all along.

I now wonder how this could impact projects. With the `NG_BUILD_LIB_LEGACY` flag, would it be wise to keep the legacy host unchanged and only remove `moduleNameToFileName` from the new implementation, so that any consequences can be mitigated by enabling `NG_BUILD_LIB_LEGACY`.